test/libyul/objectCompiler/verbatim_bug.yul

```diff
@@ -34,7 +34,7 @@ object "a" {
 //     /* "source":139:307   */
 //   dup1
 //   iszero
-//   tag_5
+//   tag_1
```

Comment:
The bug can actually be seen here directly - this replacement of tags is due to the bug in the comparison leading to an invalid tag replacement by the block deduplicator. The removal of the entire block later on is just a side-effect.